src/test/java/seedu/address/logic/commands/EditApplicantCommandTest.java

```diff
@@ -0,0 +1,182 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_APPLICANT_NAME_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_APPLICANT_NAME_BENEDICT;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_APPLICANT_NAME_CHRIS;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalIndexes.getIndexLastListing;
+import static seedu.address.testutil.TypicalListings.getTypicalListingBook;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.ListingBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.applicant.Applicant;
+import seedu.address.model.listing.Listing;
+import seedu.address.testutil.ApplicantBuilder;
+import seedu.address.testutil.ListingBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for DeleteApplicantCommand.
```

Comment:
EditApplicantCommand, not DeleteApplicantCommand

Reply:
eek thanks!